### ISSUE TITLE
IRRDB for v6 only peers

### ISSUE THREAD

**Issue by stefan6dev** [natesales/pathvector#64]
Hi @natesales,

we have some peers, there have only IPv6 Sessions to us. We set in template 
    filter-irr: true, but then the config build failed:

FATA[0002] peer has a prefix filter defined but no IPv4 prefixes

Is there any workaround to deploy only v6 only Sessions with IRRDB Filter?

Thanks!

**Comment by natesales** [comment]
Could you post an abbreviated config that causes the error? Here's a quick example that worked for me:

```yaml
asn: 65520
router-id: 192.0.2.1
cache-directory: test-cache
peers:
  Example 1:
    asn: 112
    as-set: AS112
    filter-irr: true
    neighbors:
      - 2001:db8::1
```

```bash
# pathvector generate --dry-run --config test-conf.yml
INFO[0000] Processing AS112 Example 1                   
INFO[0000] BIRD config validation passed 
```

```bash
# grep -E 'define .*_PFX.*' test-cache/AS112_EXAMPLE_1.conf -A 2
define AS112_EXAMPLE_1_PFX_v4 = [
  192.31.196.0/24,
  192.175.48.0/24
--
define AS112_EXAMPLE_1_PFX_v6 = [
  2001:4:112::/48,
  2620:4f:8000::/48
```

**Comment by stefan6dev** [comment]
In your Case, AS112 have v4 Prefixes. My downstreams have only IPv6 and than the problem will be happend

**Comment by natesales** [comment]
Got it, thanks for the clarification. I've just released v5.5.0 including commit 07dce10a1be5257aa8cd2478c33bbde346e465c5 to address this use case. Could you give it a try and let me know if it's working for you?

**Comment by stefan6dev** [comment]
Works for me, thanks 👍
